src/classes/frSetupController.cls

```diff
@@ -78,8 +109,58 @@ public with sharing class frSetupController {
 		donationFROptions.add(noneOption);
 		donorSFOptions = new List<SelectOption>();
 		donorSFOptions.add(noneOption);
+		accSFOptions = new List<SelectOption>();
+		accSFOptions.add(noneOption);
+		giftTransactionSFOptions = new List<SelectOption>();
+		giftTransactionSFOptions.add(noneOption);
 		donorFROptions = new List<SelectOption>();
 		donorFROptions.add(noneOption);
+		giftCommitmentSFOptions = new List<SelectOption>();
+		giftCommitmentSFOptions.add(noneOption);
+		giftCommitmentFROptions = new List<SelectOption>();
+		giftCommitmentFROptions.add(noneOption);
+		personAccountsEnabled = false;
+		isNonProfitOrg = false;
+
+		//Check for personaccount field on account
+        SObjectType accountType = Account.SObjectType;
+        if (frUtil.hasPersonAccountsEnabled()) {
+            personAccountsEnabled = true;
+			Map<String, Schema.SObjectField> personAccountFields = Account.sObjectType.getDescribe().fields.getMap();
+			for(String fieldName : personAccountFields.keySet()) {
+				Schema.DescribeFieldResult describe = personAccountFields.get(fieldName).getDescribe();
+				if(describe.isUpdateable()) {
+					accSFOptions.add(new SelectOption(fieldName, describe.getLabel()));
+				}
+			}
+			accSFOptions.sort();
+        }
+		//Check for GiftTransaction to see if its a NPC Org
+		if (frUtil.hasNPCobjects()) {
+			isNonProfitOrg = true;
```

Comment:
is this `isNonProfitOrg` doing anything or can it be replaced by the above call to use hasNPCObjects()?

Reply:
isNonProfitOrg is used as a reference in the frSetup page to check which mapping options it should show.